src/main/java/edu/stanford/snap/spinn3rHadoop/classifiers/README.md

```diff
@@ -0,0 +1,10 @@
+# TextVsSpamClassifier
+
+This is a simple classifier which classify Spinn3r web articles as text or spam. The classifier was built using data in folder sample-data and consists of some very simple rules described using decision tree below.
+
+## Decision tree 
+
+![alt text](https://github.com/andrazhribernik/news-search/blob/content-cleaner/src/main/java/edu/stanford/snap/spinn3rHadoop/classifiers/sample-data/TextVsSpamDecisionTree.png "Classifier decision tree image.")
```

Comment:
Would you consider changing this to a [relative link](https://github.com/blog/1395-relative-links-in-markup-files) so it is not linked to your repo?